hastexo/jobs.py

```diff
@@ -69,7 +66,9 @@ def run(self):
             stacks = Stack.objects.select_for_update().filter(
                 suspend_timestamp__isnull=False
             ).filter(
-                suspend_timestamp__lt=cutoff
+                suspend_by__isnull=False
+            ).filter(
+                suspend_by__lt=timezone.now()
```

Comment:
Hmmm maybe I'm missing something, but we don't backfill the `suspend_by` field, correct? 

So, if 
* we're not backfilling but 
* we are looking only for objects where `suspend_by` is not null, and where its content is a date in the past, 
 
doesn't that mean we effectively *don't* suspend the labs that have ceased sending keepalives less than `suspend_timeout` ago, i.e. that were last touched in the two minutes prior to migration, until the learner opens them again?

Reply:
Good point, I updated the PR.